shards/modules/physics/body.cpp

```diff
@@ -35,44 +35,53 @@ static ParamTypeMask &operator|=(ParamTypeMask &lhs, ParamTypeMask rhs) {
 struct BodyShard {
   static SHTypesInfo inputTypes() { return shards::CoreInfo::AnyType; }
   static SHTypesInfo outputTypes() { return SHBody::Type; }
-  static SHOptionalString help() { return SHCCSTR("Defines a new node"); }
+
+  static SHOptionalString inputHelp() { return DefaultHelpText::InputHelpIgnored; }
+  static SHOptionalString help() {
+    return SHCCSTR(
+        "This shard creates a physics body, conducts physics simulations on this body while updating the relavent variables tied "
+        "to this body, and creates an interface to allow other physics shards to interact with this body.");
+  }
+  static SHOptionalString outputHelp() {
+    return SHCCSTR("Outputs the physics body created by this shard as a physics object that acts as an interface for other "
+                   "physics shard to interact with the body.");
+  }
 
   static inline Type PhysicsDOFSeqType = Type::SeqOf(PhysicsDOFEnumInfo::Type);
   static inline Types PhysicsDOFTypes{PhysicsDOFEnumInfo::Type, PhysicsDOFSeqType, Type::VariableOf(PhysicsDOFSeqType)};
 
-  PARAM_PARAMVAR(_location, "Location", "The initial location, updated by physics simulation", {shards::CoreInfo::Float3VarType});
-  PARAM_PARAMVAR(_rotation, "Rotation", "The initial location, updated by physics simulation", {shards::CoreInfo::Float4VarType});
-  PARAM_VAR(_static, "Static", "Static node, persist when not activated", {shards::CoreInfo::BoolType});
-  PARAM_PARAMVAR(_enabled, "Enabled", "Can be used to toggle this node when it has static persistence",
+  PARAM_PARAMVAR(_location, "Location", "The initial location of the physics object. The variable provided in this parameter is also updated through the physics simulations conducted on this body.", {shards::CoreInfo::Float3VarType});
+  PARAM_PARAMVAR(_rotation, "Rotation", "The initial rotation of the physics object. The variable provided in this parameter is also updated through the physics simulations conducted on this body.", {shards::CoreInfo::Float4VarType});
+  PARAM_VAR(_static, "Static", "If false, the physics body will be destroyed when the shard is not called. If true, the body will persist and be included in the physics simulation even if the shard is not called.", {shards::CoreInfo::BoolType});
```

Comment:
Not sure if we used the terminology "call" with respect to shards before, "activate" is the correct one but not sure which we use in the docs